test/integration/integration.go

```diff
@@ -44,12 +43,14 @@ func skip() bool {
 	return testing.Short() || integrationTag && !inCI
 }
 
-var inCI, inGHA bool
+var inCI, inGHA, externalDB bool
 
 func init() {
-	inCI, _ = strconv.ParseBool(os.Getenv("CI"))
-	inGHA, _ = strconv.ParseBool(os.Getenv("GITHUB_ACTIONS"))
-	actuallyAct, _ := strconv.ParseBool(os.Getenv("ACT"))
+	_, inCI = os.LookupEnv("CI")
```

Comment:
Is it possible someone would set `CI=false`  or `ACT=0` or something and the behaviour would be unexpected or am I over-thinking it?